jax_unirep/utils.py

```diff
@@ -230,13 +196,34 @@ def validate_mLSTM_params(params: Dict, n_outputs):
 
 
 def load_params(folderpath: Optional[str] = None):
-    """Load params for passing to evotuning stax model."""
-    return (
-        load_mlstm_params(folderpath=folderpath),
-        (),
-        load_dense_params(folderpath=folderpath),
-        (),
-    )
+    """
+    Load params for passing to evotuning stax model.
+
+    The weights are saved as a single pickle file.
+    We did this in version 1.1 to unify how weights are stored and dumped.
+    When loaded into memory, the weights object `params`
+    will be a nested tuple of arrays and dictionaries. In order, they are:
+
+    - embedding params
+    - mLSMT1900 params (with gating weights `g*`, matrix multiplication weights `w*`, and bias `b` as keys)
+    - etc.
+
+    Loading a Pickle file can pose a security issue,
+    so if you wish to verify the MD5 of the pickles before loading them,
+    you can do so using the following block of code:
+
+    ```python
+    from jax_unirep.utils import get_weights_dir
+
+    weights_dir = get_weights_dir(folderpath=None)
+    # shell out to the system by calling on md5.
+    os.system("md5 /path/to/weights")
+    ```
```

Comment:
```suggestion
    Loading a Pickle file can pose a security issue,
    so if you wish to verify the MD5 of the pickles before loading them,
    you can do so using the following block of code:

    ```python
    from jax_unirep.utils import get_weights_dir

    weights_dir = get_weights_dir(folderpath=None)
    weights_path = weights_dir / "model_weights.pkl"
    # shell out to the system by calling on md5.
    os.system(f"md5 {str(weights_path)}")
    ```

    The return should be identical to the following:
    
        MD5 (model_weights.pkl) = 87c89ab62929485e43474c8b24cda5c8
```